.changeset/calm-pots-serve.md

```diff
@@ -0,0 +1,15 @@
+---
+'@api3/airnode-abi': patch
```

Comment:
I chose `patch`, since this is mostly fixing the previous PR which introduced the TS references.